Dockerfile

```diff
@@ -1,7 +1,7 @@
 # NOTE: this Dockerfile is purely for local development! it is *not* used for
 # the official 'concourse/concourse' image.
 
-FROM concourse/dev
+FROM concourse/dev:release-5.5
```

Comment:
nit: we also have tags for older dev images here - https://hub.docker.com/r/concourse/dev/tags
since we have branched off concourse/dev https://github.com/concourse/ci/tree/master/dockerfiles/dev